supabase/migrations/33_automate_trial_start.sql

```diff
@@ -0,0 +1,88 @@
+begin;
+
+create or replace view internal.new_free_trial_tenants as
+with hrs_by_day as (
```

Comment:
nit: `hrs` => `hours`. It's a needless abbreviation that saves two characters and can confuse a reader. Prefer to spell things out. [See also](https://google.github.io/styleguide/cppguide.html#General_Naming_Rules)